.aegir.js

```diff
@@ -18,7 +18,8 @@ function start (done) {
       (cb) => js([`${base}/10012`, `${base}/20012/ws`], true, 31012, 32012, cb),
       (cb) => js([`${base}/10013`, `${base}/20013/ws`], true, 31013, 32013, cb),
       (cb) => js([`${base}/10014`, `${base}/20014/ws`], true, 31014, 32014, cb),
-      (cb) => js([`${base}/10015`, `${base}/20015/ws`], true, 31015, 32015, cb)
+      (cb) => js([`${base}/10015`, `${base}/20015/ws`], true, 31015, 32015, cb),
+      (cb) => go([`${base}/10027`, `${base}/20027/ws`], true, 33027, 44027, cb), // we need this for circuit for now
```

Comment:
Not true, you should move the interop test to the interop suite

Reply:
Interop tests don't run in browsers right now, I thought we wanted some browser coverage?